.versionrc

```diff
@@ -0,0 +1,9 @@
+{
+  "types": [
+    {"type":"feat","section":"Features"},
+    {"type":"fix","section":"Bug Fixes"},
```

Comment:
the [tool we're using for releasing](https://github.com/googleapis/release-please) doesn't read this file format ... might actually be a good idea to add support.

But might as well drop this file for now.

Reply:
@bcoe Done. I didn't see that documented any where so I just assumed it was the same as yargs.